docs/src/starknet/invoke.md

```diff
@@ -35,10 +37,10 @@ $ sncast \
 
 ```shell
 command: invoke
-transaction_hash: 0x504f830428d0fcf462b4b814e2f67e12dfbcf3dc7847c1e36ba39d3eb7ac313
+transaction_hash: [..]
 
 To see invocation details, visit:
-transaction: https://sepolia.starkscan.co/tx/0x504f830428d0fcf462b4b814e2f67e12dfbcf3dc7847c1e36ba39d3eb7ac313
+transaction: https://sepolia.starkscan.co/tx/[..]
```

Comment:
It is very illogical that even in our examples, we show a block explorer link after executing a transaction on devnet, which makes no sense. Can we remove it? Do we have any issue to fix this?

Reply:
Looks like https://github.com/foundry-rs/starknet-foundry/issues/2488 addresses this problem. Let's not increase  scope of this PR and provide a separate one.